config/loader_test.go

```diff
@@ -49,11 +54,24 @@ this doesn't work
 			t.Errorf("error should be returned from bad toml")
 		}
 	})
+
+	t.Run("replace env key with val", func(t *testing.T) {
+		config, _ := loader.LoadString(`
+[auth]
+client_id = "{{ env "TESTVAR"}}"
+client_secret = "secret"
+`)
+
+		if config.Auth.ClientId != "test" {
+			t.Errorf("got %q want %q", config.Auth.ClientId, "test")
+		}
+	})
```

Comment:
Can you add a test where `TESTVAR` is not defined? What should it do?

Reply:
I updated the LoadString function so that it will return an error when it encounters an unknown env variable and added a test for that. Before it would just set it to an empty string which probably isn't ideal.

Reply:
Perfect, cheers!